docs/source/installation.rst

```diff
@@ -1,7 +1,7 @@
 Installation
 ============
 
-ASPIRE is based on Python 3.7, and comes with an ``environment.yml`` for reconstructing a working Conda environment to run the package.
+ASPIRE comes with an ``environment.yml`` for reconstructing a working Conda environment based on Python 3.8 to run the package.
```

Comment:
Being picky, but I think "reproducing" sounds better than "reconstructing" and also has a better connotation. Not super important either way.

Reply:
Sure, will change.